source/features/easy-toggle-files.tsx

```diff
@@ -15,14 +16,43 @@ function toggleFile(event: DelegateEvent<MouseEvent>): void {
 	}
 }
 
+function toggleCodeSearchFile(event: DelegateEvent<MouseEvent>): void {
+	const elementClicked = event.target as HTMLElement;
+	const headerBar = event.delegateTarget;
+	const toggle = $(':scope > button', headerBar)!;
+
+	// The clicked element is either the bar itself or one of its children excluding the button
+	if (elementClicked === headerBar || (elementClicked.parentElement === headerBar && elementClicked !== toggle)) {
+		toggle.dispatchEvent(new MouseEvent('click', {bubbles: true, altKey: event.altKey}));
+	}
+}
+
 function init(signal: AbortSignal): void {
 	delegate('.file-header', 'click', toggleFile, {signal});
 }
 
+function initSearchPage(signal: AbortSignal): void {
+	delegate(codeSearchHeader, 'click', toggleCodeSearchFile, {signal});
+}
+
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.hasFiles,
 		pageDetect.isGistRevision,
 	],
 	init,
+}, {
+	include: [
```

Comment:
We're gonna need 
```
asLongAs: [
	isHasSelectorSupported,
],
```

🥲 

See https://github.com/refined-github/refined-github/pull/7042